sasmodels/resolution2d.py

```diff
@@ -181,50 +181,50 @@ class Slit2D(Resolution):
 
     *q* points at which the data is measured.
 
-    *qx_width* slit width in qx
+    *q_length* slit length (long axis); assumed to be in the direction of qx
 
-    *qy_width* slit height in qy; current implementation requires a fixed
-    qy_width for all q points.
+    *q_width* slit width (short axis); assumed to be in the direction of qy; current implementation requires a fixed
+    q_width for all q points.
 
```

Comment:
why is there any assumption as to direction in q? shouldn't this be agnostic to qx and qy?

Reply:
Just subtract 90º from φ and you can switch from vertical to horizontal. Which is to say that this is a documentation problem, not a coding problem 8-)

Add something like the following to the docstring and manual:
```
For USANS, which has a vertical slit of width *q_width* sweeping through qx, add 90 degrees to the fitted
phi angle to find the orientation relative to laboratory frame.
```

Note: I might have the direction backward, and you need to subtract not add. Check before writing.

Reply:
True, but as per some of the discussion at code camp, we are trying to be more careful to ensure that PR reviews also check that any documentation which might need changing to reflect the code changes have been properly updated :-)

That said I'm not sure where the φ angle comes from or is relevant in the above?  By using slit length and width as defined by NXcanSAS in q space only there should be no laboratory frame?

Reply:
@butlerpd This is in the "2d slit smearing" perspective which does still have the laboratory frame defined, for slit-smearing anisotropic models.

Reply:
By design the φ angle rotates the pattern in the qx-qy plane independent of θ and ψ.